README.md

```diff
@@ -46,3 +46,5 @@ accept({
 ## Contributing
 
 Checkout the organization [CONTRIBUTING.md](https://github.com/react-dropzone/.github/blob/main/CONTRIBUTING.md).
+
+**NOTE** The minimum required Node version for development is v20.
```

Comment:
Why would Node.js 18 not be supported?

Reply:
I cannot recall. I think it's because the test reporter. I can check and see.

Reply:
It's the builtin lcov reporter which we use that brings this requirement.